source/administration/config-settings.rst

```diff
@@ -2995,6 +2997,30 @@ This setting determines whether team leave/join system messages are posted in th
 | This feature's ``config.json`` setting is ``"EnableDefaultChannelLeaveJoinMessages": true`` with options ``true`` and ``false`` for above settings respectively.     |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Allow Cookies for Subdomains
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting set's the Domain parameter on our cookies, which allows the browser to send the cookies to subdomains as well.
```

Comment:
To simplify, maybe we write this similarly to https://docs.mattermost.com/administration/config-settings.html#allow-banner-dismissal

I.e. No intro, just what happens when the setting is true or false.